src/popup/accounts/login.component.ts

```
@@ -79,30 +79,41 @@ export class LoginComponent implements OnInit {
         }
     }
 
-    async submit() {
-        if (this.cozyUrl == null || this.cozyUrl === '') {
-            this.platformUtilsService.showToast('error', this.i18nService.t('errorOccurred'),
-                this.i18nService.t('cozyUrlRequired'));
+    controlUrlInput(inputUrl: string): string {
+        // Prevent empty url
+        if (!inputUrl) {
+            return;
+        }
+        // Prevent email input
+        if (inputUrl.includes('@')) {
             return;
         }
```

we don't have something smarter on the cloudery or on cozy's website? I think we return something like "We don't want your email address but your cozy url instead". I don't know if we want to go this way also here

---

Nothing smarter the cloudery, the check on the email is also an `include('@')̀`. However, we could throw a specific error for the different cases